tests/nested/core/core20-fault-inject-on-refresh/task.yaml

```diff
@@ -44,7 +44,12 @@ environment:
     FAULT/kernel_reboot_refresh_gadget_assets: reboot
     TAG/gadget_reboot_refresh_gadget_assets: refresh-gadget-assets
     FAULT/gadget_reboot_refresh_gadget_assets: reboot
-    
+
+    # refresh from store might get some kernel that have an old
+    # initrd.
```

Comment:
would still be good to leave a TODO about having a FDE variant later again? also this probably should explain that the issue with old initrd is FDE not working

Reply:
I added a bit more explanation. Also added a TODO that we have to convert it to use the fakestore.